### ISSUE TITLE
Docs build is failing

### ISSUE THREAD

**Issue by ADBond** [moj-analytical-services/splink#1745]
As of [this commit](https://github.com/moj-analytical-services/splink/tree/2c3ea240f91ed01bd884d53304efba3621c71e54) (though probably unrelated) the docs deploy step is failing (see e.g. [this run](https://github.com/moj-analytical-services/splink/actions/runs/6880110600/job/18713608048)).

Error:
```
ERROR    -  Config value 'plugins': Plugin 'mkdocstrings' option 'custom_templates': The path '/home/runner/work/splink/splink/templates' isn't an existing directory.
Aborted with 1 Configuration Errors!
Error: Process completed with exit code 1.
```

Possibly a change in plugin version? But haven't looked into at all

**Comment by pawamoy** [comment]
We recently released mkdocstrings 0.24, which made the custom templates path relative to the config file directory instead of the current working directory. Looking at your code and the CI run, I'm not sure where the `templates` folder is supposed to be (I can't see it anywhere).

**Comment by ADBond** [comment]
> We recently released mkdocstrings 0.24, which made the custom templates path relative to the config file directory instead of the current working directory. Looking at your code and the CI run, I'm not sure where the `templates` folder is supposed to be (I can't see it anywhere).

Ah thanks for the info. It appears from a bit of digging that that templates directive is a phantom bit of config originating from when the docs were first set up, either leftover from trying out options or included in error.
